packages/delegation/src/__tests__/index.test.ts

```diff
@@ -1,6 +1,15 @@
 import * as ss58 from '@chainflip/utils/ss58';
-import { describe, expect, it } from 'vitest';
-import { accountIdToEthereumAddress, ethereumAddressToAccountId } from '../index.js';
+import { type PublicClient, type WalletClient } from 'viem';
+import { beforeEach, describe, expect, it, vi } from 'vitest';
+import { accountIdToEthereumAddress, DelegationSDK, ethereumAddressToAccountId } from '../index.js';
+
+vi.mock('../abis.js', () => ({
+  scUtils: `{{ SC UTILS ABI }}`,
+}));
+
+vi.mock('viem', () => ({
+  erc20Abi: '{{ ERC20 ABI }}',
+}));
```

Comment:
mocking these makes the snapshots cleaner but requires an additional test suite for the ABIs to ensure they are "tested"